docs/core/docker/publish-as-container.md

```diff
@@ -0,0 +1,349 @@
+> ```
+
+> [!TIP]
+> Depending on the type of app you're containerizing, the command-line switches (options) might vary. For example, the `/t:PublishContainer` argument is only required for non-web .NET apps, such as `console` and `worker` templates. For web templates, replace the `/t:PublishContainer` argument with `-p:PublishProfile=DefaultContainer`. For more information, see [.NET SDK container builds, issue #141](https://github.com/dotnet/sdk-container-builds/issues/141).
+
+The command will output similar to the following:
+
+```dotnetcli
+Determining projects to restore...
+  All projects are up-to-date for restore.
+  DotNet.ContainerImage -> .\Worker\bin\Release\net7.0\linux-x64\DotNet.ContainerImage.dll
+  DotNet.ContainerImage -> .\Worker\bin\Release\net7.0\linux-x64\publish\
+  Pushed container 'dotnet-worker-image:1.0.0' to registry 'docker://'
+```
+
+This command compiles your worker app to the *publish* folder and pushes the container to your local docker registry.
+
+## Configure container image
+
+You can control many aspects of the generated container through MSBuild properties. In general, if you could use a command in a _Dockerfile_ to set some configuration, you can do the same via MSBuild.
+
+> [!NOTE]
+> The only exception to this are `RUN` commands, due to the way containers are built, those cannot be emulated. If you need this functionality, you will need to use a _Dockerfile_ to build your container images.
+
+> [!IMPORTANT]
+> Only Linux containers are currently supported.
+
+### `ContainerBaseImage`
+
+This property controls the image used as the basis for your image. By default, we will infer the following values for you based on the properties of your project:
+
+* if your project is self-contained, we use the `mcr.microsoft.com/dotnet/runtime-deps` image as the base image
+* if your project is an ASP.NET Core project, we use the `mcr.microsoft.com/dotnet/aspnet` image as the base image
+* otherwise we use the `mcr.microsoft.com/dotnet/runtime` image as the base image
+
+We infer the tag of the image to be the numeric component of your chosen `TargetFramework` - so a `.net6.0` project will use the `6.0` tag of the inferred base image, a `.net7.0-linux` project will use the `7.0` tag, and so on.
+
+If you set a value here, you should set the fully-qualified name of the image to use as the base, including any tag you prefer:
+
+```xml
+<ContainerBaseImage>mcr.microsoft.com/dotnet/runtime:6.0</ContainerBaseImage>
+```
+
+### `ContainerRegistry`
+
+This property controls the destination registry - the place that the newly-created image will be pushed to.
+
+Be default, we push to the local Docker daemon (annotated by `docker://`), but for this release you can specify any _unauthenticated_ registry. For example:
```

Comment:
By GA this will change - an empty `ContainerRegistry` will mean pushing to docker. This is due to some fixes around our registry parsing code to become more in-line with the golang specs.

Reply:
So what change should be made here?

Reply:
Maybe just remove `(annotated by `docker://)` so that the intent remains the same across versions?